index.html

```diff
@@ -154,6 +154,26 @@ <h3><dfn>Frequency</dfn></h3>
       thermal throttling helps cool the [=processing unit=] down when it gets too hot by lowering the frequency.
     </p>
   </section>
+  <section>
+    <h3><dfn>Sampling Frequency</dfn> and <dfn>Reporting Frequency</dfn></h3>
+    <p>
+      The [=sampling frequency=] for a [=platform collector=] is defined as a frequency at which the user agent
+      obtains telemetry readings from the underlying platform.
+    </p>
+    <p>
+      The [=reporting frequency=] for a concrete [=ComputePressureObserver=] object is defined as a frequency
+      at which the update callback of this object will be invoked.
+    </p>
+    <p>
+      A [=ComputePressureObserver=] object cannot access new readings at a higher rate than the user agent
+      obtains them from the underlying platform, therefore the [=reporting frequency=] can never exceed the [=sampling frequency=].
+    </p>
+    <p>
+      The [=reporting frequency=] can vary based on the difference in values between readings at [=sampling frequency=].
+      If between two consecutive samplings the values of telemetry readings are unchanged,
```

Comment:
Sounds a bit weird, "If the values between two con.... are unchanged"

Reply:
but its not really about the values though.. it depends on whether the values result in changing buckets